docs/source/tutorial/package_a_project.md

```diff
@@ -1,46 +1,65 @@
-# Package a project
+# Package a Kedro project
 
 This section explains how to build your project documentation, and how to bundle your project into a Python package.
 
+
 ## Add documentation to your project
 
-To generate project-specific documentation, run `kedro build-docs` in the project's root directory. Kedro builds the resulting HTML files in `docs/build/html/`. To browse the documentation generated, open `docs/build/html/index.html` in your browser, or run `kedro build-docs --open` to open the documentation automatically after it builds.
+Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) and creates a `docs` directory that builds a basic template for project-specific documentation. We recommend that you add your project-specific documentation as markdown in `docs/source`
 
-The `build-docs` command creates documentation based on the code structure of your project. Documentation includes any [`docstrings`](https://datacamp.com/community/tutorials/docstrings-python) defined in your code.
+If you want to customise your documentation beyond the basic template, refer to the [Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html) for details of how to extend `docs/source/conf.py`.
 
-Kedro uses the [Sphinx framework](https://www.sphinx-doc.org), so if you want to customise your documentation, please refer to `docs/source/conf.py` and the [corresponding section of the Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html).
+Once you have added any documentation you need, run the following from the project root directory:
+
+```bash
+kedro build-docs --open
+```
+
+The HTML documention is built to `docs/build/html` and opens automatically in a browser tab.
+
+```{note}
+The `build-docs` command creates documentation based on the code structure of your project. Documentation includes any [`docstrings`](https://datacamp.com/community/tutorials/docstrings-python) defined in your code.
+```
 
 
 ## Package your project
 
-To package your project, run the following in your project's root directory:
+To package your project, run the following in your project root directory:
 
 ```bash
 kedro package
 ```
 
-Kedro builds the package into the `dist/` folder of your project, and creates one `.egg` file and one `.whl` file, which are [Python packaging formats for binary distribution](https://packaging.python.org/).
+Kedro builds the package into the `dist` folder of your project, and creates one `.egg` file and one `.whl` file, which are [Python packaging formats for binary distribution](https://packaging.python.org/).
```

Comment:
I think we need an admonition that highlights the difference between this and pipeline micropacking as well as a link to that documentation in case they're looking for that.